PROJECT.md

```diff
@@ -0,0 +1,291 @@
+request should have:
+
+- (Required) One or more [`area/*`](https://github.com/moby/moby/labels?q=area) labels
+- (Required) An [issue type](#types-of-issues) or [`kind/*`](https://github.com/moby/moby/labels?q=kind) label for pull requests
+  
+Additional labels can provide more clarity:
+
+- Zero or more [`needs/*`](https://github.com/moby/moby/labels?q=needs) labels to indicate missing items
+- Zero or more [`impact/*`](https://github.com/moby/moby/labels?q=impact) labels to identify major areas impacted by the change
+- One [`exp/*`](https://github.com/moby/moby/labels?q=exp) label to estimate the difficulty of the work
+- A [`priority/*`](https://github.com/moby/moby/labels?q=priority) label (required for `Bug` types)
+
+> [!TIP] 
+> Most labels should have a description, read this for for information or
+> post a [discussion in Q&A](https://github.com/moby/moby/discussions/categories/q-a)
+
+### Project backlog
+
+This project maintains a backlog -— a holding area for issues and pull requests
+that have been approved by maintainers but are not yet scheduled for release.
+When a work item is assigned to a milestone, it moves into planning, indicating
+that it has been scheduled for a release.  
+
+Backlog items will have an [`accepted`](https://github.com/moby/moby/issues?q=sort%3Aupdated-desc+is%3Aopen+label%3Aaccepted) label.
+
+## :pushpin: Projects
+
+GitHub [projects](https://github.com/moby/moby/projects?query=is%3Aopen) are
+used to support [project](#-workflow-and-tracking) operations. These boards can
+be used to track the progress of changes, expected features in a release and
+insight into the project roadmap. Read the project documentation for more
+information about the views and fields being used.
+
+## :speech_balloon: Discussions
+
+[GitHub discussions](https://github.com/moby/moby/discussions) serve a community
+forum for this project. Community members can ask questions, share updates and
+have open-ended discussions on project-related topics.
+
+Sometimes, an issue or pull request may not be the appropriate medium for what
+is essentially a discussion. In such cases, the issue or PR will either be
+converted to a discussion or a new discussion will be created.
+
+If you believe this conversion was made in error, please express your concerns
+in the new discussion thread. If necessary, a reversal to the original issue or
+PR format can be facilitated.
+
+## :gear: Workflow automation
+
+To help expedite common operations, avoid errors and reduce toil some workflow
+automation is used by the project. This can include:
+
+- Stale issue or pull request processing
+- Auto-labeling actions
+- Auto-response actions
+- Label carry over from issue to pull request
+
+### Exempting an issue/PR from stale bot processing
+
+The stale item handling is configured in the [repository](link-to-config-file).
+To exempt an issue or PR from stale processing you can:
+
+- Add the item to a milestone
+- Add the `state/frozen` label to the item
```

Comment:
While we've got the hood open, would we be open to renaming this "neverstale" or something like that? :innocent:

("begonebot")